plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityAnger.java

```diff
@@ -3,115 +3,51 @@
 import com.denizenscript.denizen.objects.EntityTag;
 import com.denizenscript.denizencore.objects.core.DurationTag;
 import com.denizenscript.denizencore.objects.Mechanism;
-import com.denizenscript.denizencore.objects.ObjectTag;
-import com.denizenscript.denizencore.objects.properties.Property;
-import com.denizenscript.denizencore.objects.properties.PropertyParser;
 import org.bukkit.entity.Bee;
-import org.bukkit.entity.Entity;
 import org.bukkit.entity.PigZombie;
 
-public class EntityAnger implements Property {
+public class EntityAnger extends EntityProperty<DurationTag> {
 
-    public static boolean describes(ObjectTag entity) {
-        if (!(entity instanceof EntityTag)) {
-            return false;
-        }
-        Entity bukkitEntity = ((EntityTag) entity).getBukkitEntity();
-        return bukkitEntity instanceof PigZombie
-                || bukkitEntity instanceof Bee;
-    }
-
-    public static EntityAnger getFrom(ObjectTag entity) {
-        if (!describes(entity)) {
-            return null;
-        }
-        else {
-            return new EntityAnger((EntityTag) entity);
-        }
-    }
+    // <--[property]
+    // @object EntityTag
+    // @name anger
+    // @input DurationTag
+    // @description
+    // Controls the remaining anger time of a PigZombie or Bee.
+    // -->
 
-    public static final String[] handledMechs = new String[] {
-            "anger"
-    };
-
-    public EntityAnger(EntityTag entity) {
-        this.entity = entity;
+    public static boolean describes(EntityTag entity) {
+        return entity.getBukkitEntity() instanceof Bee
+                || entity.getBukkitEntity() instanceof PigZombie;
     }
 
-    EntityTag entity;
-
     @Override
-    public String getPropertyString() {
-        return new DurationTag((long) getAnger()).identify();
+    public DurationTag getPropertyValue() {
+        if (getEntity() instanceof PigZombie pigzombie) {
+            return new DurationTag((long) pigzombie.getAnger());
+        }
+        else if (getEntity() instanceof Bee bee) {
+            return new DurationTag((long) bee.getAnger());
+        }
+        return null;
     }

```

Comment:
Just a nitpick, but don't need to have an `instanceof` check for bees here, as the entity is necessarily either a bee or a zombie piglin (I.e. can just `as(Bee.class)` like you did in the mechanism)